File: Proyectos-Ejercicios/AplicacionMapa/app/src/main/java/com/example/aplicacionmapa/retrofit/RetrofitServices.java
package com.example.aplicacionmapa.retrofit;

import com.example.aplicacionmapa.datamaps.MapasItem;
import retrofit2.Response;
import retrofit2.http.GET;

interface RetrofitServices {

    @GET("sitios")
    suspend fun getlugares(): Response<MapasItem>

}

File: Proyectos-Ejercicios/RetrofitSampleClase/app/src/main/java/com/estech/retrofitsample/retrofit/RetrofitServices.kt
package com.estech.retrofitsample.retrofit

import com.estech.retrofitsample.modelos.Respuesta
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.POST

interface RetrofitServices {

    // ES PARA PONER AQUI TODAS LAS FUNCIONES DE LOS ENDPOINTS DE LAS API


    @GET("character")
    suspend fun getallpersonajes(): Response<Respuesta>

    @POST("batch/{id}/add-item")

}